jwt/api_jws.py

```diff
@@ -128,7 +128,7 @@ def encode(self,
         return b'.'.join(segments)
 
     def decode(self,
-               token,  # type: str
+               jwt,  # type: str
```

Comment:
@jpadilla, @mark-adams: I might be wrong, but this looks like it used to be `jws` before, as can be seen in here: https://github.com/jpadilla/pyjwt/commit/aed7305680b0ca6cf9846cd755b420f4dfccee22#diff-971ea3149c0d11912490c16fc9f982a9L120